### PR TITLE
Updates to the RVC start page to include 100% height info

### PR DESCRIPTION
**Pull request by sundeep1982**

### CODE REVIEW

File: Docs_14_0/Set-100Percent-Height-With-RVC.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+### How to set 100% height on the new Report Viewer 2016 control
+The new Report Viewer 2016 control is optimized for HTML5 Standards mode pages and works on all modern browsers. In the past with the old RVC control when you set the 100% height property it worked even if none of the ancestors had height specified. This behavior has changed in HTML5. When you set this property on the new RVC control, it will work correctly only if the parent element has a defined height, i..e not a value of auto, or all the ancestors of RVC have 100% height too.
```
Comment (jtarquino) [inline Docs_14_0/Set-100Percent-Height-With-RVC.md]: standard I think

Comment (jtarquino) [inline Docs_14_0/Set-100Percent-Height-With-RVC.md]: also i.e. single dot

File: Docs_14_0/Get-Started-With-RVC.md [hunk 2]
```diff
@@ -111,3 +111,6 @@ To make use of the 2016 Report Viewer Control in an existing project add the con
 ### Page is rendering in intranet compatability mode
 
 The 2016 Report Viewer Control is designed to be used with modern browsers, the control may not function if browsers render the web page in an IE compatibility mode. Intranet sites may require a '<meta http-equiv="X-UA-Compatible" content="IE=edge" />' tag to override setting which encourage rendering intranet pages in compatibility mode.
+
+### How to set 100% height on the new Report Viewer 2016 control
+[For detailed instructions click here](http://htmlpreview.github.io/?https://github.com/Microsoft/Reporting-Services/blob/master/Docs_14_0/Set-100Percent-Height-With-RVC.html)
```
Comment (jtarquino) [inline Docs_14_0/Get-Started-With-RVC.md]: I would do For detailed instructions [click here]

File: Docs_14_0/Set-100Percent-Height-With-RVC.md (hunk 1 above)
Comment (sundeep1982) [inline Docs_14_0/Set-100Percent-Height-With-RVC.md]: it is standards mode. https://developer.mozilla.org/en-US/docs/Quirks_Mode_and_Standards_Mode

Comment (sundeep1982) [inline Docs_14_0/Set-100Percent-Height-With-RVC.md]: fixing it

File: Docs_14_0/Get-Started-With-RVC.md (hunk 2 above)
Comment (sundeep1982) [inline Docs_14_0/Get-Started-With-RVC.md]: fixing it

**Pull request merged**